vue.md

```diff
@@ -0,0 +1,51 @@
+# Vue
+
+**Prerequisite knowledge**: js, es6, single-page application, framework, npm
+
+## Intro
+Vue.js (commonly referred to as Vue; pronounced /vjuː/, like view) is an open-source JavaScript framework for building user interfaces and single-page applications. Vue.js features an incrementally adoptable architecture that focuses on declarative rendering and component composition. Advanced features required for complex applications such as routing, state management and build tooling are offered via officially maintained supporting libraries and packages. [[Source](https://en.wikipedia.org/wiki/Vue.js)]
+
+## Classification / acceptance questions
+
+### Apprentice aka _Hello World_
+* I can instantiate Vue object
+* I can create a component
+* I understand Vue component lifecycle
+* I can use Vue template syntax to render component HTML
+
+### Intermediate aka _My first SPA_
+**Prerequisite knowledge**: sass
+* I can use conditional rendering (`v-if` vs `v-show`)
+* I can render a list (`v-for`)
+* I can style a component
+* I can pass pass parameters to a component (`props`)
+* I can emit and handle component events (`$emit`)
+* I can use computed properties and watchers
+* I can handle form input using `v-model`
+* I can pass content inside another component using slots
+* I can dynamically switch between components using `<component>`
+* I can use filters for data transformations in templates
+
+### Advanced aka _TODO SPA_
+* I can use an external plugin
+* I can use routes to navigate in the application using Vue router
+* I can use Vuex state manager to simplify data flow in the application
+* I can validate Prop Types
+* I can use scoped slots to access data from the child component
+* I can use `<transition>` component to make the application more smooth
+* I can prepare production build
+
+### Master aka _Under the hood_
+**Prerequisite knowledge**: debugger
+* I can create my own plugin
+* I can debug Vue application using Vue.js devtools
```

Comment:
Vue devtools can be useful right from the beginning. What is more, I think they can make it easier to understand what's happening under the hood.

Reply:
I personally wasn't using such tools at low levels. What is your experience?

Reply:
I started using it right away. I'm using dev tools for other frameworks as well and at the very beginning I use them to track the data flow and better understand what is happening under the hood. But this is my default way of learning, first understand, then write :)